platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -171,17 +235,78 @@ public Schema getSchema(CFMetaData metadata) {
         }
     }
 
+    public Schema getSchema() {
+        return this.actualSchema;
+    }
+
+    private static SSTableReadsListener newReadCountUpdater() {
```

Comment:
why is this needed?

Reply:
To satisfy the input arguments of `ssTableReader.getScanner`. We need to filter the data using  `DataRange` and `ColumnFilter` when getting the scanner. Although there are several types of `getScanner`, but the one with both filters requires `SSTableReadsListener` as the last input argument.